test/e2e/default_bridged_network_test.go

```diff
@@ -157,8 +159,8 @@ func nodeReadyConditionStatus(nodeName string) (corev1.ConditionStatus, error) {
 func waitForNodesReady() {
 	time.Sleep(5 * time.Second)
 	for _, node := range nodes {
-		Eventually(func() (corev1.ConditionStatus, error) {
+		EventuallyWithOffset(1, func() (corev1.ConditionStatus, error) {
 			return nodeReadyConditionStatus(node)
-		}, 60*time.Second, 1*time.Second).Should(Equal(corev1.ConditionTrue))
+		}, 30*time.Minute, 10*time.Second).Should(Equal(corev1.ConditionTrue))
```

Comment:
30 minutes is too much. If we need to wait more than 5 minutes for a node to come back, it is a bug.